OpenAL can't link to 64-bit Qt. Disable Qt for now
--- openal-soft-1.16.0/utils/alsoft-config/CMakeLists.txt.~1~	2015-07-21 13:57:59.672133697 +0300
+++ openal-soft-1.16.0/utils/alsoft-config/CMakeLists.txt	2015-07-21 13:58:12.011182924 +0300
@@ -3,27 +3,27 @@
 include_directories("${alsoft-config_BINARY_DIR}")
 
 # Need Qt 4.8.0 or newer for the iconset theme attribute to work
-find_package(Qt4 4.8.0 COMPONENTS QtCore QtGui)
-if(QT4_FOUND)
-    include(${QT_USE_FILE})
-
-    set(alsoft-config_SRCS  main.cpp
-                            mainwindow.cpp
-        )
-
-    set(alsoft-config_UIS  mainwindow.ui)
-    QT4_WRAP_UI(UIS ${alsoft-config_UIS})
-
-    set(alsoft-config_MOCS  mainwindow.h)
-    QT4_WRAP_CPP(MOCS ${alsoft-config_MOCS})
-
-    add_executable(alsoft-config ${alsoft-config_SRCS} ${UIS} ${RSCS} ${TRS} ${MOCS})
-    target_link_libraries(alsoft-config ${QT_LIBRARIES})
-    set_target_properties(alsoft-config PROPERTIES RUNTIME_OUTPUT_DIRECTORY ${OpenAL_BINARY_DIR})
-
-    install(TARGETS alsoft-config
-            RUNTIME DESTINATION bin
-            LIBRARY DESTINATION "lib${LIB_SUFFIX}"
-            ARCHIVE DESTINATION "lib${LIB_SUFFIX}"
-    )
-endif()
+#find_package(Qt4 4.8.0 COMPONENTS QtCore QtGui)
+#if(QT4_FOUND)
+#    include(${QT_USE_FILE})
+#
+#    set(alsoft-config_SRCS  main.cpp
+#                            mainwindow.cpp
+#        )
+#
+#    set(alsoft-config_UIS  mainwindow.ui)
+#    QT4_WRAP_UI(UIS ${alsoft-config_UIS})
+#
+#    set(alsoft-config_MOCS  mainwindow.h)
+#    QT4_WRAP_CPP(MOCS ${alsoft-config_MOCS})
+#
+#    add_executable(alsoft-config ${alsoft-config_SRCS} ${UIS} ${RSCS} ${TRS} ${MOCS})
+#    target_link_libraries(alsoft-config ${QT_LIBRARIES})
+#    set_target_properties(alsoft-config PROPERTIES RUNTIME_OUTPUT_DIRECTORY ${OpenAL_BINARY_DIR})
+#
+#    install(TARGETS alsoft-config
+#            RUNTIME DESTINATION bin
+#            LIBRARY DESTINATION "lib${LIB_SUFFIX}"
+#            ARCHIVE DESTINATION "lib${LIB_SUFFIX}"
+#    )
+#endif()
